### PR TITLE
Phoneme

### PR DESCRIPTION
**Pull request by mariaalouisaa**
**NOT ready to be merged!**

But beginning the message so I don't forget what I did...

If you are testing the deploy follow child onboarding to see whats been done - but when you get to a dead end manually go to the phoneme-game/correct-guess path to see the layout page being used with different props sent.

- Made a VERY basic child landing page that has a link to what will be the phoneme game
- The landing page has a place holder for their name to be added later
- Started on a phoneme game layout component that has:
          ~ two TTS components that accept text from a received property
          ~ a home icon that links to the landing page
          ~ a start icon with a use state for the star count (i guess this needs to be change to 
          the local storage custom hook)
           ~ a button that need the functionality finished  but i tried to put in some pseudo code
- Centred the h1 to be able to position home and stars on the learners pages

### CODE REVIEW

**Review by vasystus — APPROVED** [review]
Amazing, ready to merge!

File: components/LearnerOnboardingLayout.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ export default function LearnerOnboardingLayout({
       <LearnerOnboardingProgressBar stepNumber={stepNumber} totalSteps={5} />
       <Tts>{ttsTitle}</Tts>
       <div>{children}</div>
-      <Link href={`/learner-onboarding/${nextStep}`}>
+      <Link
```
Comment (PJSalter) [inline components/LearnerOnboardingLayout.js:20]: Great work on the Link for next step ✨

File: components/PhonemeLayout.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// need to figure out what props we will send for the gameplay - the word broken into an array for the tiles makes sense. Then we can map through and use the same template regarless of how long the word is :)
+
+export default function LearnerOnboardingLayout({
+  firstText,
+  secondText,
+  context,
+}) {
+  // Will want to use and update local storage with the stars
+  const [stars, setStars] = useState(0);
+  const [buttonText, setButtonText] = useState("Go!");
```
Comment (PJSalter) [inline components/PhonemeLayout.js:17]: Very clear code ✨

File: components/Styled-Components/GameNext.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+const GameNext = styled.button`
```
Comment (PJSalter) [inline components/Styled-Components/GameNext.js:3]: well done on the styling components styling 🎨

File: pages/phoneme-game/first-step.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import PhonemeLayout from "../../components//PhonemeLayout";
+
+export default function PhonemeFirstStep() {
+  const html = (
+    <>
+      <p>HERE IS A GAME PLACEHOLDER</p>
+      {["c", "a", "t"].map((letter) => (
```
Comment (PJSalter) [inline pages/phoneme-game/first-step.js:7]: Very clear, really enjoyed reading the code ✨

Comment (PJSalter) [inline pages/phoneme-game/first-step.js:7]: 😸  great choice on  start of spelling the animal for cat 🐱

**Review by PJSalter — APPROVED** [review]

**Pull request merged**